src/collective/jekyll/symptoms.py

```diff
@@ -207,51 +200,45 @@ def help(self):
             mapping={'maximum': self.maximum}
         )
 
-    EMPTY = dict(
-        description=_(u"Description does not have content."),
-        summary=_(u"Summary does not have content."),
-    )
+    EMPTY = _(u"Description does not have content.")
 
-    TOO_LONG = dict(
-        description=_(
+    TOO_LONG = _(
             u"The description counts ${word_count} significant words "
             u"(longer than 3 letters). "
             u"It should have at most ${maximum} significant words."
-        ),
-        summary=_(
-            u"The summary counts ${word_count} significant words "
-            u"(longer than 3 letters). "
-            u"It should have at most ${maximum} significant words."
-        ),
-    )
+        )
 
     def _update(self):
         maximum = self.maximum
         word_count = countWords(self.context.Description())
-        if 'summary' in self.context.Schema()['description'].widget.label:
-            key = 'summary'
-        else:
-            key = 'description'
```

Comment:
We should keep the distinction between description and summary. The description field is still named summary in the "Page" content type.

Reply:
It is not the only place where keeping the distinction is needed.